### PR TITLE
Added SWAGGER_FILE env var

### LINKED ISSUE #1695: Environment variable to import json or yaml
| Q                                   | A
| ----------------------------------- | -------
| Bug or feature request?             |   feature request
| Which Swagger/OpenAPI version?      | 
| Which Swagger-Editor version?       | 
| How did you install Swagger-Editor? | docker
| Which browser & version?            | Chrome Version 62.0.3202.89 (Official Build) (64-bit)
| Which operating system?             |  Ubuntu

### Current Behavior

Able to install docker image and run using `docker run -d -p 80:8080 swaggerapi/swagger-editor`

### Possible Solution

Should be able to import json on start
`docker run -d -p 80:8080 -e SWAGGER_JSON='./swagger.json' swaggerapi/swagger-editor`

### Context

Trying to start fresh with provided json instead of starting editor and importing everytime. I am limited to docker only run

### PR DESCRIPTION
**Pull request by codeasashu**
Added support for `SWAGGER_FILE` in docker images of swagger-editor.

### Description

Earlier it wasn't possible to use a local `json` or `yaml` swagger schema file to be used inside docker via env vars. Issue #1695 describes this in details. This is the PR that fixes it.

### Motivation and Context

Fixes  #1695

### How Has This Been Tested?

1. Pulled the repo and created a local docker image
```
git pull https://github.com/swagger-api/swagger-editor
## After making my changes
docker build -t swaggerapi/swagger-editor .
docker run -d -p 80:8080 -v $PWD:/tmp -e SWAGGER_FILE=/tmp/swagger.yaml swaggerapi/swagger-editor
```
seems to load correct `yaml` definition that only exists on my local machine in current dir.

### My PR contains... 

- [x] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [x] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [x] My changes require a change to the project documentation.
- [x] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docker-run.sh [hunk 1]
```diff
@@ -2,8 +2,19 @@
...
-INDEX_FILE=/usr/share/nginx/html/index.html
+NGINX_ROOT=/usr/share/nginx/html
+INDEX_FILE=$NGINX_ROOT/index.html
 
 # TODO: this is empty but we'll be adding configuration values here
 
-exec nginx -g 'daemon off;'
\ No newline at end of file
+## Adding env var support for swagger json
+if [[ -f $SWAGGER_JSON ]]; then
```
Comment (z3niths) [inline docker-run.sh]: I think we can use `SWAGGER_FILE` or `DEFAULT_FILE` instead since it's working with both `json` and `yaml`

Comment (codeasashu) [reply docker-run.sh]: yes, makes sense

Comment (codeasashu) [reply docker-run.sh]: Fixed in 675812c

**Comment by shockey** [comment]
(LGTM besides my review question!)

File: index.html [hunk 2]
```diff
@@ -51,6 +51,7 @@
   window.onload = function() {
     // Build a system
     const editor = SwaggerEditorBundle({
+      url: "https://petstore.swagger.io/v2/swagger.json",
```
[the remark this answers is not in the document]
Comment (codeasashu) [reply index.html]: petstore example is what I see by default whenever I start the swagger editor. I have seen the default json included with the source code, to be petstore. If `url` is not a required attrib, you can kindly skip it. I am outside rn, and wont be able to change the source code. Feel free to edit them for me pls

**Comment by jm3** [comment]
Any chance we can merge this?

**Comment by Abir-H** [comment]
Can we get this merged please

**Comment by cjuif** [comment]
Hi,
Would be great if these changes could be merged.
Thanks,

**Comment by evgeniy-khist** [comment]
@tim-lai, it looks like the merge commit [f93e1ea](https://github.com/swagger-api/swagger-editor/commit/f93e1ea8fce8f8e927310324ee56cfeb172030fb) into master contains an error.
In the commit [update index.html per reviewer comment](https://github.com/swagger-api/swagger-editor/pull/2072/commits/400063b7cc1c3ee33258f6e99dd1dae33fa216e5) you removed the line
```
url: "https://petstore.swagger.io/v2/swagger.json",
```
So, the following block in the [docker-run.sh](https://github.com/swagger-api/swagger-editor/blob/master/docker-run.sh) is useless:
```
if [[ -f $SWAGGER_FILE ]]; then
  cp -s $SWAGGER_FILE $NGINX_ROOT
  REL_PATH="/$(basename $SWAGGER_FILE)"
  sed -i "s|https://petstore.swagger.io/v2/swagger.json|$REL_PATH|g" $INDEX_FILE
fi
```
`sed` will not replace anything and Pet Store will remain the default.

**Pull request merged**